src/ui/Button/__tests__/__snapshots__/Button.spec.js.snap

```diff
@@ -4,7 +4,6 @@ exports[`Button should do a snapshot test of the default Button DOM 1`] = `
 <button
   className=" sendbird-button  sendbird-button--primary sendbird-button--big"
   disabled={false}
-  onClick={[Function]}
```

Comment:
This is becuase we are not passing default onclick fn to fallback to, can we add it?